doc/calcurse.1.txt

```diff
@@ -143,19 +143,21 @@ are marked "('also interactively')".
 
 *-c* 'file', *--calendar* 'file'::
   ('also interactively') Specify the calendar file to use. The default
-  calendar is *~/.calcurse/apts* (see <<_files,FILES>>). If 'file' is not an
-  absolute path name, it is interpreted relative to the current working
-  directory. The option has precedence over *-D*.
+  calendar is *$XDG_DATA_HOME/apts* (*~/.local/share/calcurse/apts*)
+  (see <<_files,FILES>>).  If 'file' is not an absolute path name, it is
+  interpreted relative to the current working directory. The option has
+  precedence over *-D*.
 
 *-C* 'dir', *--confdir* 'dir'::
   ('also interactively') Specify the configuration directory to use. If not
-  specified, the default directory is *~/.calcurse/*. See <<_files,FILES>> for
-  the interaction with *-D*.
+  specified, the default directory is *$XDG_CONFIG_HOME/calcurse/*
+  (*~/.config/calcurse/*). See <<_files,FILES>> for the interaction with *-D*.
```

Comment:
We should definitely mention that if `~/.calcurse/` exists, then it takes precedence over `~/.config/calcurse/`. Same goes for the other options. Also, while rebasing, it would be great if you could change the commit subjects to use imperative mood ("Change documentation to [...]").